DEVELOPING_OPAL/demos/src/main/scala/org/opalj/fpcf/analyses/AndroidAnalysisDemo.scala

```diff
@@ -0,0 +1,103 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package fpcf
+package analyses
+
+import java.net.URL
+
+import com.typesafe.config.ConfigValueFactory
+
+import org.opalj.ai.domain
+import org.opalj.ai.fpcf.properties.AIDomainFactoryKey
+import org.opalj.br.analyses.BasicReport
+import org.opalj.br.analyses.Project
+import org.opalj.br.analyses.ProjectAnalysisApplication
+import org.opalj.br.analyses.cg.InitialEntryPointsKey
+import org.opalj.br.android.AndroidManifestKey
+import org.opalj.br.fpcf.PropertyStoreKey
+import org.opalj.tac.cg.CFA_1_0_CallGraphKey
+import org.opalj.util.PerformanceEvaluation.time
+import org.opalj.util.Seconds
+
+object AndroidAnalysisDemo extends ProjectAnalysisApplication {
+
+    override def title: String = "Generate Callgraph for Android App"
+
+    override def description: String = "Parses AndroidManifest.xml to determine entry points"
+
+    override def doAnalyze(
+        project:       Project[URL],
+        parameters:    Seq[String],
+        isInterrupted: () => Boolean
+    ): BasicReport = {
+        val manifestPath = parameters.collectFirst {
+            case param if param.startsWith("-manifest=") => param.split("=")(1)
```

Comment:
Please override `analysisSpecificParametersDescription` to document this parameter